### PR TITLE
load_word_topics() returns value

### PR DESCRIPTION
**Pull request by bhargavvader**
@tmylk, @piskvorky , with reference to #764 .

`load_word_topics()` now returns `word topics` which is assigned by `train` to `self.wordtopics`.

Before, there were two variables; `word_topics` and a `wordtopics`, and only the `wordtopics` variable was being used elsewhere; so I removed `word_topics`.

Is that ok?

### CODE REVIEW

**Comment by tmylk** [comment]
Rename wordtopics to word_topics.

Also should there be a meaningful error like 'run train or load_word_topcs before showing topics' in https://github.com/bhargavvader/gensim/blob/6fd1ecbfe41d7ab2adc17d20179d467e986f477a/gensim/models/wrappers/ldamallet.py#L244

**Comment by bhargavvader** [comment]
Will do. Also, this PR doesn't seem to have a travis build happening, any reason?

**Comment by dsquareindia** [comment]
oh renaming to `word_topics` would break some other things actually....

**Comment by tmylk** [comment]
@bhargavvader Well spotted about Travis - other PRs are working ok. Let's see what happens on your next commit.

**Comment by bhargavvader** [comment]
@dsquareindia , exactly what else would break? Not if we change all the `wordtopics` to `word_topics` right?

**Comment by dsquareindia** [comment]
@bhargavvader the `mallet2ldamodel` would fail and also the support for `ldamallet` in topic coherence would fail. Tests pass because tests for one of them hasn't been added and for the other one hasn't been merged. I'll change the spelling in my PR but could you please change it in `coherencemodel.py`? Also a quick `git grep` would help.

**Comment by bhargavvader** [comment]
Cool, on it.

**Comment by dsquareindia** [comment]
yeah @bhargavvader I'll make both the changes in my pr once this gets merged. Thanks!

**Comment by bhargavvader** [comment]
@tmylk , @piskvorky , can you have a look?
Also, about the warning, I've just used `logger.info` now. Is that ok?

File: gensim/models/wrappers/ldamallet.py [hunk 1]
```diff
@@ -242,7 +242,9 @@ def show_topics(self, num_topics=10, num_words=10, log=False, formatted=True):
         return shown
 
     def show_topic(self, topicid, topn=10):
-        topic = self.wordtopics[topicid]
+        if self.word_topics is None:
+            logger.info("Run train or load_word_topics before showing topics.")
```
Comment (tmylk) [inline gensim/models/wrappers/ldamallet.py]: warn instead of info

Comment (bhargavvader) [inline gensim/models/wrappers/ldamallet.py]: Yup, did.

**Comment by tmylk** [comment]
Thanks for the PR. It's a useful fix!

**Comment by piskvorky** [comment]
-1 on renaming `wordtopics` to `word_topics`. This breaks backward compatibility. At least make an alias.

File: gensim/models/wrappers/ldamallet.py [hunk 2]
```diff
@@ -199,10 +199,10 @@ def load_word_topics(self):
                 if token not in word2id:
                     continue
                 tokenid = word2id[token]
-                wordtopics[int(topic), tokenid] += 1.0
-        logger.info("loaded assigned topics for %i tokens", wordtopics.sum())
-        self.wordtopics = wordtopics
+                word_topics[int(topic), tokenid] += 1.0
+        logger.info("loaded assigned topics for %i tokens", word_topics.sum())
         self.print_topics(15)
```
Comment (piskvorky) [inline gensim/models/wrappers/ldamallet.py]: This print makes no sense without the `self.wordtopics` assignment.

Comment (bhargavvader) [inline gensim/models/wrappers/ldamallet.py]: You're right - the print was there in the code earlier and I removed the `self.wordtopics` assignment so now it doesn't make sense.

**Comment by bhargavvader** [comment]
@piskvorky , how exactly would one make an alias?
And what exactly do you mean by breaking the older code? I had checked fir occurrences of `wordtopics` in the repo and couldn't find anything I didn't change. Or do you mean tutorials and the like?

I'll make another PR for that and for the unnecessary print statement. Is there anything else you think should change?

**Comment by piskvorky** [comment]
I mean, we use it in our code, in commercial projects :)

I think it's safe to assume some other people depend on it too, in the same way. Unless it's unavoidable, we try to be backward-compatible with our changes.

Alias = two variables referring to the same object, i.e. `x = y = some_var`.

**Comment by bhargavvader** [comment]
Ok, makes sense. 

As for making an alias in this case, would you mean adding a line such as `self.wordtopics = self.word_topics = word_topics` or such whenever an assignment is being made?

**Comment by piskvorky** [comment]
Yes, assign `self.wordtopics = self.word_topics`, with a big fat comment explaining why this alias is there.

**Pull request merged**